### PR TITLE
Further VHcc changes. Important variables to be stored

### PR DESCRIPTION
**Pull request by ValVau**

### CODE REVIEW

**Comment by valsdav** [comment]
HI! Can you rebase your branch on the current master? There are conflicts

**Comment by ValVau** [comment]
Hi. Hope it now fits better.

File: pocket_coffea/lib/jets.py [hunk 1]
```diff
@@ -23,13 +24,26 @@ def load_jet_factory(params):
         return cloudpickle.load(fin)
         
 
-def jet_correction(params, events, jets, factory, jet_type,  year, cache):
-    return factory[jet_type][year].build(
-        add_jec_variables(jets, events.fixedGridRhoFastjetAll), cache
-    )
+def jet_correction(params, events, jets, factory, jet_type, chunk_metadata, cache):
```
Comment (valsdav) [inline pocket_coffea/lib/jets.py:27]: I don't understand what git is doing here, this is already the version we have in main https://github.com/PocketCoffea/PocketCoffea/blob/main/pocket_coffea/lib/jets.py#L27

Can you make sure to include only the changes in the last part of the file please?

Comment (ValVau) [reply pocket_coffea/lib/jets.py:27]: I will try my best. I don't understand neither, what git does here

**Comment by ValVau** [comment]
Should work now

**Comment by valsdav** [comment]
Good to go! thanks

**Pull request merged**